### ISSUE TITLE
Akaari's Shadowstrike causes Shadow Blades which double dips on Deeper Daggers

### ISSUE THREAD

**Issue by Mystler** [SimCMinMax/WoW-BugTracker#691]
**Game Version:**
> 9.0.2 beta Build 36037

**Description:**
Akaari's Shadowstrikes also trigger Shadow Blades attacks, despite technically not being CP generators. With Shadow Blades and Deeper Daggers up, Shadow Blades attacks also seem to deal 30% increased damage, which causes the Shadow Blades attack for the Akaari Strike to double dip.

The only other Shadow damage attack that works with Shadow Blades is Gloomblade. In comparison, here it does not get the 30% increase so only the base spell is amped by Deeper Daggers. Apparently, some manual exclusion has happened here.

TL;DR: Akaari's Shadowstrike should either not trigger Shadow Blades, or the corresponding Blades attacks should not double dip on Deeper Daggers like for Gloomblade.

**Logs, Images, Videos**
![image](https://user-images.githubusercontent.com/161936/94372571-26a5e980-00ff-11eb-92cf-9ec14857ea5f.png)

Examples during Deeper Daggers. Both Shadow Blades attacks are 65% of the spell causing it (`0.5*1.3`), which is a double dip in case of the Shadow Shadowstrike.

**Comment by Mystler** [comment]
Closing. Akaari strike is physical again.
